### DISCUSSION TITLE
Having an issue with Camera head

### DISCUSSION THREAD

**Discussion by Astronomia-maker** [ifm/ifm3d#428] [Q&A]
Hello, 

I am using 2 camera heads (O3R222) with connect to VPU 800. [Port0,port1:> 2d, 2d] and [port2, port3:> 3d,3d] with respected camera. But on port3 camera always in ERROR mode even though i set to RUN mode using iVA or python API and raise an Error :- Invalid device configuration. 

Both cameras dont have any direct human interference. But it second camera [port3] says head shut down due to eye protection.

Software stack:- 
VPU firmware:- 1.0.14
ifm3dpy==1.3.3
python= 3.10.11

I also have attached screenshot for reference.  

![Screenshot 2023-10-19 215051](https://github.com/ifm/ifm3d/assets/110673255/dcae878a-3c0c-487b-abd9-4c75d3bb823f)

**Comment by lola-masson** [comment]
The eye safety shutdown might indicate a defective camera. Try re-updating the firmware and see if the diagnostic message disappears (sometime it can have to do with the camera head being improperly updated). If the diagnostic message does not disappear after re-updating, then you will be able to exchange it.

**Comment by Astronomia-maker** [comment]
I have another O3R225. But it's running on old firmware (0.16.xx) with vpu800. So can i just replace vpu with newer firmware just to find out that camera is defective or not. Because i have already reupated firmware and problem still exist.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by lola-masson** [reply]
You won't be able to connect the camera to the old firmware, because there is no backward compatibility between 0.16.xx and 1.x.x. If the issue still persists, then get in contact with your sales engineer at ifm to exchange the defective camera.
